lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,27 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+$cms-blue: rgb(0, 159, 230); // NOTE - eventually, this set of blues should be synced with CMS.  
```

Comment:
This is all the same as the blues used in Studio, the only difference is the the base blue is called $cms-blue instead of calling it $blue and forcing all LMS blues over to CMS with this change. Using $blue-l2 on the LMS would be the same hex value as the $blue-l2 for CMS today. With this change, people can start using the lights / darks of this blue set instead of m-blue-* (which is the only option otherwise.) It felt wrong to keep adding more m-blue-* variables without doing some of the cleanup. We'd still need a cleanup task to remove all m-blue-* to use this syntax, as well as all update all uses of $blue, but this shouldn't impact any colors currently in the LMS. 

Does that make sense?